### PR TITLE
get rid of extra node integration tests

### PR DESCRIPTION
**Pull request by arroyc**
re structuring some node integration tests to get rid of over testing and improving build and release time

### CODE REVIEW

**Comment by arroyc** [comment]
node integration test time reduced 
image
master branch node integration test duration
image

improvement 22.5%

File: tests/Oryx.Integration.Tests/Node/NodeOtherEndtoEndTests.cs [hunk 1]
```diff
@@ -247,48 +247,6 @@ public async Task CanBuildAndRunNodeApp_OnSecondBuild_AfterNotZippingNodeModules
                 });
         }
 
-        [Theory]
-        [InlineData("10.10")]
-        [InlineData("10.14")]
-        public async Task CanBuildAndRun_NodeApp(string nodeVersion)
-        {
-            // Arrange
-            var appName = "webfrontend";
```
Comment (gpcastro) [inline tests/Oryx.Integration.Tests/Node/NodeOtherEndtoEndTests.cs]: is this app being tested elsewhere?

File: tests/Oryx.Integration.Tests/Node/NodeOtherEndtoEndTests.cs [hunk 2]
```diff
@@ -573,52 +530,6 @@ public async Task Node_CreateReactAppSample()
...
         }
 
-        [Theory]
-        [InlineData("8.11")]
-        [InlineData("8.12")]
-        [InlineData("10.1")]
-        [InlineData("10.10")]
-        [InlineData("10.14")]
-        [InlineData("12")]
-        public async Task Node_CreateReactAppSample_zippedNodeModules(string nodeVersion)
```
Comment (gpcastro) [inline tests/Oryx.Integration.Tests/Node/NodeOtherEndtoEndTests.cs]: why is this being removed? Do we have zipped react apps tested elsewhere?

File: tests/Oryx.Integration.Tests/Node/NodeTestUsingZippedNodeModules.cs [hunk 3]
```diff
@@ -23,8 +23,11 @@ public NodeTestUsingZippedNodeModules(ITestOutputHelper output, TestTempDirTestF
         }
 
         [Theory]
-        [MemberData(nameof(TestValueGenerator.GetZipOptions_NodeVersions), MemberType = typeof(TestValueGenerator))]
-        public async Task CanBuildAndRunNodeApp_UsingZippedNodeModules(string compressFormat, string nodeVersion)
+        //[MemberData(nameof(TestValueGenerator.GetZipOptions_NodeVersions), MemberType = typeof(TestValueGenerator))]
```
Comment (gpcastro) [inline tests/Oryx.Integration.Tests/Node/NodeTestUsingZippedNodeModules.cs]: this test helps to make sure we have the compression tools in every image. I'm fine with removing it as long as we have a test that uses compressed tar-gz node_modules for every node version.

File: tests/Oryx.Integration.Tests/Node/NodeOtherEndtoEndTests.cs (hunk 1 above)
Comment (arroyc) [reply tests/Oryx.Integration.Tests/Node/NodeOtherEndtoEndTests.cs]: yes

**Pull request merged**